build.sbt

```diff
@@ -141,13 +150,13 @@ lazy val argonaut = project
   .settings(moduleName := "finch-argonaut")
   .settings(allSettings)
   .settings(libraryDependencies += "io.argonaut" %% "argonaut" % "6.1")
-  .dependsOn(core, test % "test")
+  .dependsOn(core, test % "test", `json-test` % "test")
```

Comment:
I'm pretty sure the JSON implementation projects don't need a dependency on `test` now.